pubspec.yaml

```diff
@@ -16,13 +16,13 @@ platforms:
   linux:
 
 dependencies:
-  http: ^1.0.0
+  http: ^0.13.5
```

Comment:
Why do you need to downgrade http? 
I tried your branch in my repo and because of this version solving failed.